packages/aws-cdk/lib/init.ts

```diff
@@ -292,26 +293,34 @@ async function postInstallTypescript(canUseNetwork: boolean, cwd: string) {
   const command = 'npm';
 
   if (!canUseNetwork) {
-    print(`Please run ${colors.green(`${command} install`)}!`);
+    warning(`Please run ${colors.green(`${command} install`)}!`);
     return;
   }
 
   print(`Executing ${colors.green(`${command} install`)}...`);
   try {
     await execute(command, ['install'], { cwd });
   } catch (e) {
-    throw new Error(`${colors.green(`${command} install`)} failed: ` + e.message);
+    warning(`${command} install failed: ` + e.message);
+    warning(`Please run ${command} install`);
```

Comment:
`${command} install` - Isn't that what you just attempted to run and just failed? Would it succeed if the user run it again?

Reply:
it is, it would succeed if it was a 500, and wouldn't otherwise.
I think it's reasonable to drop it for now and add it later if it's helpful.